reana_server/scheduler.py

```diff
@@ -60,6 +66,9 @@ def check_predefined_conditions():
             #              DiskPressure, PIDPressure,
             for condition in node.get("status", {}).get("conditions", {}):
                 if not condition.get("status"):
+                    logging.info(
+                        f"Workflow could not be started due to predefined condition: {condition}"
```

Comment:
Cannot start workflow 60515eff-33ca-468a-a55c-6cf72d72af5d due to predefined condition foo, bar and baz.